app/players/elves/druid.py

```diff
@@ -0,0 +1,19 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str
+                 ) -> None:

```

Comment:
```

    def __init__(
             self,
             nickname: str,
             musical_instrument: str,
             favourite_spell: str
     ) -> None:
```

Reply:
There is an error in your proposed solution Flake8 "Closing bracket does not match visual indentation"